Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -355,8 +359,22 @@ void Renderer::DrawDebugText()
 		case SCALE_4X:
 			res_text = "4x";
 			break;
+		case SCALE_5X:
+			res_text = "5x";
+			break;
+		case SCALE_6X:
+			res_text = "6x";
+			break;
+		case SCALE_7X:
+			res_text = "7x";
+			break;
+		case SCALE_8X:
+			res_text = "8x";
+			break;
+		default:
+			res_text = (g_ActiveConfig.iEFBScale - 3) + "x";
```

Comment:
Isn't this formulation valid for 3x IR and higher? So all of them can just fall back to default.